jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -407,7 +409,7 @@ void checkAccess(Class<?> clazz) throws IllegalAccessException {
 		 * @throws IllegalAccessException If the member is not accessible.
 		 * @throws IllegalAccessError If a handle argument or return type is not accessible.
 		 */
-		private void checkAccess(Class<?> targetClass, String name, int memberModifiers, MethodHandle handle, boolean skipAccessCheckPara) throws IllegalAccessException {
+		private void checkAccess(Class<?> targetClass, Class<?> definingClass, String name, int memberModifiers, MethodHandle handle, boolean skipAccessCheckPara) throws IllegalAccessException {
```

Comment:
I really dislike changing the meaning of the variable name here as it makes it hard to review whether changes were deliberate or not.